tests/network-ovn

```diff
@@ -2091,6 +2091,12 @@ ovn_leases_tests() {
 
         lxc delete -f c1
 
+        echo "==> Check OVN external addresses show on lxc network list-allocations"
+        ovn_ext_ipv4="$(lxc network get ovn-virtual-network volatile.network.ipv4.address)"
+        ovn_ext_ipv6="$(lxc network get ovn-virtual-network volatile.network.ipv6.address)"
+        lxc network list-allocations --format csv | grep -e "/1.0/networks/ovn-virtual-network,${ovn_ext_ipv4}"\
```

Comment:
Please test that in 2 separated steps as otherwise either of the 2 patterns matching will result in a successful command pipeline.